js/functions.ts

```diff
@@ -0,0 +1,14 @@
+const lessOrEqual = (string:string, maxLength:number) => string.length <= maxLength;
```

Comment:
Либо уточнить функцию

```suggestion
const enum Default {
	MAX_STRING_LENGTH = 100,
}

const lessOrEqual = (string: { length: number }, maxLength: number = Default.MAX_STRING_LENGTH) => string.length <= maxLength;
```